addons/mail/static/src/js/activity.js

```diff
@@ -254,6 +233,16 @@ const BasicActivity = AbstractField.extend({
                 $popover.addClass('o_mail_activity_feedback').attr('tabindex', 0);
                 $popover.find('#activity_feedback').focus();
                 self._bindPopoverFocusout($(this));
```

Comment:
forgot to remove function

Reply:
Good catch, so that part is not tested, good to know...